Serialization.md

```diff
@@ -0,0 +1,184 @@
+
+```
+  <Simulation id="XMVCOCRQwky6ttpq36MONA" name="Simple">
+    <BuildConfiguration>
+      <Molecules id="dJRCM06S7UG-ZbT3H6GNAQ" name="Simple" bbVersion="0">
+        <Builders>
+          <MoleculeBuilder id="jrKhlw58IkeRn6kWO6UYFA" name="C1" icon="Drug" mode="Logical" containerType="Other" isFloating="1" quantityType="Drug" isXenobiotic="1" defaultStartFormula="jjQg_fgKKkOJLrMEuyOC5g">
+            <Children>
+              <Parameter id="QR9mTPJCvU69y38G4q-big" name="pKa_pH_WS_sol_K7" description="Supporting parameter for calculation of pKa- and pH- dependent solubilty scale factor at refPH" persistable="0" isFixedValue="0" dim="Dimensionless" quantityType="Parameter" formula="CompoundAcidBase_PKSim.PARAM_pKa_pH_WS_sol_K7" buildMode="Property" visible="0" canBeVaried="0" />
+              <Parameter id="UliE_UU0c0uWAKtlM8vFqg" name="pKa_pH_WS_sol_F3" description="Supporting parameter for calculation of pKa- and pH- dependent solubilty scale factor at refPH" persistable="0" isFixedValue="0" dim="Dimensionless" quantityType="Parameter" formula="CompoundAcidBase_PKSim.PARAM_pKa_pH_WS_sol_F3" buildMode="Property" visible="0" canBeVaried="0" />
+              .
+              .
+              .
+              </Children>
+            <UsedCalculationMethods>
+              <UsedCalculationMethod category="DistributionCellular" calculationMethod="Cellular partition coefficient method - PK-Sim Standard" />
+              .
+              .
+              .
+            </UsedCalculationMethods>
+          </MoleculeBuilder>
+```
+
+
+The only case where things are a bit more complicated is the Formula Cache. If you open the xml segment of the Formula Cache in a .pkml file you will see something like the following:
+
+```
+ <FormulaCache>
+          <Formulas>
+            <Formula id="nyGiXWaSW0OYhQJ__lyZhg" name="Concentration_formula" dim="Concentration (molar)" formula="M/V">
+              <Paths>
+                <Path path="0" as="1" dim="2" />
+                <Path path="3" as="4" dim="5" />
+              </Paths>
+            </Formula>
+            <Formula id="Jv7-T-BLjEKUKypb2n2YIA" name="PLnL_zu_MFormula" dim="Concentration (molar)" formula="1e15/6.02214179e23" />
+            <Formula id="lU8mEpGab0-unqhumPvr6g" name="PLFormula" dim="Concentration (molar)" formula="dilutionFactor*S_PL*PLnL_zu_M">
+              <Paths>
+                <Path path="6" as="7" />
+                <Path path="8" as="9" />
+                <Path path="10" as="11" />
+              </Paths>
+            </Formula>
+            .
+            .
+            .
+         </Formulas>
+```
+The above excerpt is a simple PK-Sim project, and more specifically the [S1_concentrBased.pkml](https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/tests/PKSim.Tests/Data/S1_concentrBased.pkml) that is part of the test data of PKSim codebase.
```

Comment:
"simple PK-Sim project" - would replace this with something like "simple simulation created in PK-Sim and exported to pkml"
(because PK-Sim projects are databases and not xmls/pkmls)

Reply:
corrected